build/patches/expose-crypto-option-aes-128-sha1-80.patch

```diff
@@ -115,3 +115,66 @@ index af5f195d98..30cb504f1b 100644
    native_crypto_options.srtp.enable_encrypted_rtp_header_extensions =
        Java_Srtp_getEnableEncryptedRtpHeaderExtensions(jni, j_srtp);
    native_crypto_options.sframe.require_frame_encryption =
+diff --git a/sdk/objc/api/peerconnection/RTCConfiguration.mm b/sdk/objc/api/peerconnection/RTCConfiguration.mm
+index eeb94939f3..b7e57dac32 100644
+--- a/sdk/objc/api/peerconnection/RTCConfiguration.mm
++++ b/sdk/objc/api/peerconnection/RTCConfiguration.mm
+@@ -127,6 +127,8 @@
+                                                      .enable_gcm_crypto_suites
+              srtpEnableAes128Sha1_32CryptoCipher:config.crypto_options->srtp
+                                                      .enable_aes128_sha1_32_crypto_cipher
++             srtpEnableAes128Sha1_80CryptoCipher:config.crypto_options->srtp
++                                                     .srtpEnableAes128Sha1_80CryptoCipher
+           srtpEnableEncryptedRtpHeaderExtensions:config.crypto_options->srtp
+                                                      .enable_encrypted_rtp_header_extensions
+                     sframeRequireFrameEncryption:config.crypto_options->sframe
+diff --git a/sdk/objc/api/peerconnection/RTCCryptoOptions.h b/sdk/objc/api/peerconnection/RTCCryptoOptions.h
+index b465bb5a73..ccea9bf8bd 100644
+--- a/sdk/objc/api/peerconnection/RTCCryptoOptions.h
++++ b/sdk/objc/api/peerconnection/RTCCryptoOptions.h
+@@ -33,6 +33,13 @@ RTC_OBJC_EXPORT
+  * other ciphers get preferred.
+  */
+ @property(nonatomic, assign) BOOL srtpEnableAes128Sha1_32CryptoCipher;
++/**
++ * If set to true, the (potentially insecure) crypto cipher
++ * SRTP_AES128_CM_SHA1_80 will be included in the list of supported ciphers
++ * during negotiation. It will only be used if both peers support it and no
++ * other ciphers get preferred.
```

Comment:
The comment is from the AES GCM ciphers, right? Because I don't think it's correct.

Reply:
No, I think it's adapted from the srtpEnableAes128Sha1_32CryptoCipher comment above.

```
/**
 * If set to true, the (potentially insecure) crypto cipher
 * SRTP_AES128_CM_SHA1_32 will be included in the list of supported ciphers
 * during negotiation. It will only be used if both peers support it and no
 * other ciphers get preferred.
 */
@property(nonatomic, assign) BOOL srtpEnableAes128Sha1_32CryptoCipher;
```

Reply:
Okay, then the comment should be removed. `crypto_options.h` says:

```c
    // If set to true, the (potentially insecure) crypto cipher
    // SRTP_AES128_CM_SHA1_32 will be included in the list of supported ciphers
    // during negotiation. It will only be used if both peers support it and no
    // other ciphers get preferred.
    bool enable_aes128_sha1_32_crypto_cipher = false;

    // The most commonly used cipher. Can be disabled, mostly for testing
    // purposes.
    bool enable_aes128_sha1_80_crypto_cipher = true;
```

Reply:
I removed the comments about security and negotiation. OK now?